cluster/addons/dns/skydns-rc.yaml.in

```diff
@@ -19,6 +19,9 @@ spec:
         version: v11
         kubernetes.io/cluster-service: "true"
     spec:
+{% if grains['cloud'] is defined and grains['cloud'] in [ 'vsphere', 'photon-controller' ] %}
```

Comment:
You linked me back to this same PR - ?

Sorry for the chaos, you've just ended up in territory that has outgrown
the current implementation...  This yaml.in file didn't start as a jinja
template, it started as an example, which grew into a jinja template, but
in the meantime, other uses of it are not jinja...

On Sun, May 1, 2016 at 2:56 PM, Alain Roy <notifications@github.com> wrote:

> In cluster/addons/dns/skydns-rc.yaml.in
> <https://github.com/kubernetes/kubernetes/pull/24124#discussion_r61693813>
> :
>
> > @@ -19,6 +19,9 @@ spec:
> >          version: v11
> >          kubernetes.io/cluster-service: "true"
> >      spec:
> > +{% if grains['cloud'] is defined and grains['cloud'] in [ 'vsphere', 'photon-controller' ] %}
>
> Before you roll back, could you look at this pull request and see if it
> addresses the problem?
> #24124 <https://github.com/kubernetes/kubernetes/pull/24124>
> It looks like it could be merged, but it hasn't been--I'm not sure why.
>
> The issue this was solving wasn't about knowing which nameservers to
> contact, it was about actually being able to contact them: without enabling
> the use of the host network, we were unable to communicate with the DNS
> server.
>
> Ultimately, this change was meant to only affect the vSphere and Photon
> Controller implementations. Clearly I didn't do that correctly. Is there a
> way we can keep the underlying change without breaking other
> implementations?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/24124/files/5699d51efadb41b6c010f45672c4844d84de0b84#r61693813>
>